specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2024-02-01/vaults.json

```diff
@@ -0,0 +1,2296 @@
+            "Failure",
+            "CriticalFailure",
+            "PartialSuccess"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceMoveState",
+            "modelAsString": true
+          }
+        },
+        "backupStorageVersion": {
+          "description": "Backup storage version",
+          "enum": [
+            "V1",
+            "V2",
+            "Unassigned"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "BackupStorageVersion",
+            "modelAsString": true
+          }
+        },
+        "publicNetworkAccess": {
+          "description": "property to enable or disable resource provider inbound network traffic from public clients",
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "PublicNetworkAccess",
+            "modelAsString": true
+          }
+        },
+        "monitoringSettings": {
+          "description": "Monitoring Settings of the vault",
+          "$ref": "#/definitions/MonitoringSettings"
+        },
+        "restoreSettings": {
+          "description": "Restore Settings of the vault",
+          "$ref": "#/definitions/RestoreSettings"
+        },
+        "redundancySettings": {
+          "description": "The redundancy Settings of a Vault",
+          "type": "object",
+          "properties": {
+            "standardTierStorageRedundancy": {
+              "description": "The storage redundancy setting of a vault",
+              "enum": [
+                "Invalid",
+                "LocallyRedundant",
+                "GeoRedundant",
+                "ZoneRedundant"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "standardTierStorageRedundancy",
+                "modelAsString": true
+              }
+            },
+            "crossRegionRestore": {
+              "description": "Flag to show if Cross Region Restore is enabled on the Vault or not",
+              "enum": [
+                "Enabled",
+                "Disabled"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "crossRegionRestore",
+                "modelAsString": true
+              }
+            }
+          }
+        },
+        "securitySettings": {
+          "description": "Security Settings of the vault",
+          "$ref": "#/definitions/SecuritySettings"
+        },
+        "secureScore": {
+          "description": "Secure Score of Recovery Services Vault",
+          "type": "string",
+          "enum": [
+            "None",
+            "Minimum",
+            "Adequate",
+            "Maximum"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "SecureScoreLevel",
+            "modelAsString": true
+          }
+        },
+        "bcdrSecurityLevel": {
```

Comment:
>bcdrSecurityLevel [](http://example.com/codeflow?start=9&length=17)

BCDR seems like jargon, or perhaps overly abbreviated for a general audience -- can we use a more easily understandable name, or at least document it clearly?